app/assets/javascripts/dashboard.js

```diff
@@ -9,10 +9,13 @@ function auto_refresh(){
 
   if (element[0]) {
     refresh_timeout = setTimeout(function(){
-      if ($(".auto-refresh").hasClass("on")) {
+      var autoRefreshIsOn = $(".auto-refresh").hasClass("on");
+      var hasFocus = tfm.document.doesDocumentHasFocus();
```

Comment:
imfo, ```tfm.doc.hasFocus``` makes more sense (as you are already in the context of document?)

Reply:
So I did think about it and my conclusion was that inside the `react_app` stack people would use it as well:
```js
import { hasFocus as docHasFocus } from '../common/document';
```

So I wasn't sure what I prefer, does it still make sense for you to call it `hasFocus`?

Or maybe just for the `tfm` to put it under `tfm.doc`?